server/views/partials/layout.njk

```diff
@@ -224,6 +224,7 @@
         });
       }
     </script>
+    <script src="/javascripts/external.min.js"></script>
```

Comment:
yes fair point, I'm currently using webpack for this so maybe we put in backlog (as it would be good if we ported our js build step to this), in meantime I'll just get it to generate a hash for cache busting